### DISCUSSION TITLE
Neovim plugin for Oxlint

### DISCUSSION THREAD

**Discussion by soulsam480** [oxc-project/oxc#7488] [General]
I tried to use oxlint in neovim with https://github.com/mfussenegger/nvim-lint but it doesn't use the language server. also there's no config in https://github.com/neovim/nvim-lspconfig. so I tried to make it work.

https://github.com/soulsam480/nvim-oxlint
- it's little crude and mostly based on https://github.com/esmuellert/nvim-eslint
- I haven't tested the code actions, but I assume it'll work

**Comment by nrayburn-tech** [comment]
Is https://github.com/oxc-project/coc-oxc what you're looking for (a bit outdated, but that will be fixed eventually)?  Sorry I'm not really familiar with neovim.

**Reply by soulsam480** [reply]
it didn't work, so had to make one
